app/lib/geordi-logger.coffee

```diff
@@ -4,12 +4,21 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
   constructor: (@state, @geordi) ->
 
   @tokens = ['zooHome', 'zooTalk', 'zooniverse/gravity-spy']
+  @GEORDI_WARNINGS_TIMEOUT_INTERVAL = 5 * 60 * 1000 # only warn about Geordi down once every 5 mins, to avoid console spam
```

Comment:
Yeah - i figured it was signalling something constant. Just seems strange that there is no difference between the declarations on lines 6-7 but the naming convention changes. I don't really mind the caps but it should be consistent with the rest of the repo.